Makefile

```diff
@@ -12,6 +12,8 @@ TOOLS_OUT := $(ROOT_OUT)/tools
 DOC_OUT := $(ROOT_OUT)/doc
 BUILD_VERSION ?=
 BUILD_TAG ?=
+DESTDIR ?=
+SYSROOTDIR ?=
```

Comment:
How do you use this `SYSROOTDIR` variable? Is this something that you expect people to set via the command-line?

I'm asking but I find it odd to have a weak assignment in this Makefile but the variable is not being used at all (neither passed on when calling other Makefiles, nor exported as a variable) so this seems useless to have it here.

Reply:
I supposed to use it by passing it to make command or exporting it in shell in which make is being executed.

I think you are right, it does not have to be in the top Makefile, it's redundant and useless, will drop it in V3, and I am also going to rename it to SYSROOT, will send a V3 soon.